src/commands/bootstrap/index.ts

```diff
@@ -512,89 +876,111 @@ const bootstrap = async (props: CommonProps) => {
       throw new Error(`Creating a Next.js project failed: ${String(error)}.`);
     }
 
-    if (finalOptions.auth === 'auth.js') {
+    if (options.auth === 'auth.js') {
       // Generate AUTH_SECRET using openssl
-      const authSecret = execSync('openssl rand -base64 33').toString().trim();
+      const devAuthSecret = execSync('openssl rand -base64 33')
```

Comment:
this would likely not work on windows, right?

Reply:
Good call. I've just checked how the Auth.js CLI does it (https://github.com/nextauthjs/cli/blob/main/commands/secret.js#L9) and copied.

I also thought about doing a `npx auth secret` but I think that's too complex.